assets/src/edit-story/components/dropzone/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useRef, useLayoutEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useDropZone from './useDropZone';
+
+const DropZoneComponent = styled.div`
+	display: inherit;
```

Comment:
`display: inherit` seems very uncommon. Why is it needed?

Reply:
In page navigation context, the dropzone's display should be `flex` (as its parent), in some other context probably not, thus the `inherit`. Thoughts?

Reply:
I'm not sure that would work like this. You can inherit `display:flex`, but what about `flex-direction` and very many other flex/grid/etc CSS properties? I this for this reason there was `display:contents` introduced sometime ago. But not sure if that ever became a reality and in what form.

Reply:
Good point. I'll remove this, we can handle any specific display needs per case basis later if they should come.